### PR TITLE
Add links for DALI installations to docs

### PR DESCRIPTION
**Pull request by klecki**
## Description:
Add links to the NGC containers.
Adjust the NVIDIA Optimized Deep Learning Framework, powered by Apache MXNet container name.
Add links from support matrix to respective doc pages
Adjust some of CUDA 10 -> CUDA 11 information.

### Affected modules and functionalities:
Docs

### Key points relevant for the review:
Docs

### Tests:

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [x] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [x] RST
  - [ ] Jupyter
  - [ ] Other
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [ ] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by klecki** [comment]
!build

File: docs/installation.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ Prerequisites
 .. _compatibility link : https://docs.nvidia.com/deploy/cuda-compatibility/index.html#enhanced-compat-minor-releases
 
 1. Linux x64.
-2. |driver link|_ supporting `CUDA 10.0 <https://developer.nvidia.com/cuda-downloads>`__ or later (i.e., 410.48 or later driver releases).
+2. |driver link|_ supporting `CUDA 11.0 <https://developer.nvidia.com/cuda-downloads>`__ or later (i.e., 450.80.02 or later driver releases).
 3. (Optional) One or more of the following deep learning frameworks:
 
   - |mxnet link|_ ``mxnet-cu100`` or later.
```
Comment (JanuszL) [inline docs/installation.rst:24]:
```suggestion
  - |mxnet link|_ ``mxnet-cu110`` or later.
```
?

**Review by JanuszL — APPROVED** [review]

File: docs/installation.rst (hunk 1 above)
Comment (klecki) [reply docs/installation.rst:24]: All the framework version could be looked at. Either we should remove them or update them.

Comment (JanuszL) [reply docs/installation.rst:24]: Maybe we can remove the version as we test with a fixed version but NGC containers test with the latest. Also, this info doesn't tell anything meaningful to the users. Usually, you get the version you like most and try to use DALI with it, and so far rarely anyone complains.

Comment (klecki) [reply docs/installation.rst:24]: Yeah, I already removed the mention of a DLFW container version, I can remove those as well. No one in their right mind would be checking those versions, and we pretty much work with anything recent.

Comment (JanuszL) [reply docs/installation.rst:24]: Cannot agree more.

**Comment by klecki** [comment]
!build

File: docs/installation.rst [hunk 2]
```diff
@@ -6,34 +6,41 @@ Prerequisites
...
+  - |tf link|_
+  - |pddl link|_
 
 
 DALI in NGC Containers
 ----------------------
 
-DALI is preinstalled in the TensorFlow, PyTorch, and MXNet containers in versions 18.07 and
-later on `NVIDIA GPU Cloud <https://ngc.nvidia.com>`_.
+DALI is preinstalled in the `TensorFlow <https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tensorflow>`_,
```
Comment (JanuszL) [inline docs/installation.rst:35]: Please add the PaddlePaddle container here.

**Review by JanuszL — APPROVED** [review]

File: docs/installation.rst (hunk 2 above)
Comment (klecki) [reply docs/installation.rst:35]: done

**Comment by klecki** [comment]
!build

**Pull request merged**